src/components/Table/GridTable.tsx

```diff
@@ -232,6 +232,8 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = an
   const { tableState } = api;
 
   tableState.onRowSelect = onRowSelect;
+  // useEffect(() => {
+  // }, [tableState, rows]);
```

Comment:
Should remove this; I really want to drive this through a `useEffect` to remove the mobx/react warnings, but for some reason it doesn't "just work" like the other "move mutating the observable to useEffects" have.